webcompat/static/css/development/components/report-form.css

```diff
@@ -15,6 +15,19 @@
     text-align: center;
     justify-content: center;
     flex-wrap: wrap;
+    margin-top: 1em;
+  }
+
+  .wc-ReportForm .wc-UploadForm {
+    border: dashed 1px rgb(204, 204, 204);
+    border-radius: 20px;
+  }
+
+  .wc-ReportForm .r-Grid-cell {
```

Comment:
I'm not a big fan to this kind of selector. Each component is independ to avoid side effect. We can add ne wrapper to handle it.